startup_script.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+project="clgcporg2-100"
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+echo "$(basename $0)"
+gcloud compute --project=$project firewall-rules create tcp --direction=INGRESS --priority=1000 --network=default --action=ALLOW --rules=tcp:22 --source-ranges=0.0.0.0/0
+gcloud compute --project=$project firewall-rules create default-puma-server --direction=INGRESS --priority=1000 --network=default --action=ALLOW --rules=tcp:9292 --source-ranges=0.0.0.0/0 --target-tags=puma-server
+gcloud beta compute routes create default --project=$project --network=default --priority=1000 --destination-range=0.0.0.0/0 --next-hop-gateway=default-internet-gateway
+gcloud compute project-info add-metadata --metadata-from-file=ssh-keys=/home/fallgeratoor/gcloud-script/TXT.pub
```

Comment:
this path `/home/fallgeratoor/gcloud-script` is specific only for your workstation and script will be broken if you use the another host. Replace this path by something more portable which use well known path, for example:

$HOME/.ssh/appuser.pub